packages/next/build/webpack/config/blocks/css/index.ts

```diff
@@ -284,16 +285,6 @@ export const css = curry(async function css(
             sideEffects: false,
             // CSS Modules are activated via this specific extension.
             test: regexCssModules,
-            // CSS Modules are only supported in the user's application. We're
-            // not yet allowing CSS imports _within_ `node_modules`.
-            issuer: {
```

Comment:
Removal of this line is causing `next-transpile-module` plugin to fail with Next@13.0.1. Is this something that we can restore?

Reply:
@ijjk Does it mean Next.js is now allowing natively SCSS/CSS import from modules located in `node_modules`? O_O